iceoryx_posh/test/moduletests/test_popo_base_subscriber.cpp

```diff
@@ -146,20 +145,6 @@ TEST_F(BaseSubscriberTest, ReceiveForwardsErrorsFromUnderlyingPort)
     // ===== Cleanup ===== //
 }
 
```

Comment:
Test not needed anymore. The propagation of the "error" type is tested elsewhere. Adapting this test would essentially replicate this as we now propagate NO_CHUNK_AVAILABLE.